pkg/cmd/serve/cmd.go

```diff
@@ -19,6 +32,14 @@ func NewServeCommand(env *environments.Env) *cobra.Command {
 			err := env.CreateServices()
 			if err != nil {
 				glog.Fatalf("Unable to initialize environment: %s", err.Error())
+			} else {
+				// Get the build information:
+				buildInfo, err := showBuildInfo()
+				if err != nil {
+					glog.Errorf("unable to retrieve buildinfo")
+				} else {
+					glog.Infof("Current Go Version: %s. Current VSC and Operating System information: %v. Main package path: %s.", buildInfo.GoVersion, buildInfo.Settings, buildInfo.Path)
```

Comment:
I think we should be more specific about what we want. There's some information that we don't really need.
The current implementation dumps entire structs of information. I'd like to limit it to:
* Build commit SHA
* Architecture
* Operating system
* Build time
* VCS type
* Go version used